lib/jekyll/reader.rb

```diff
@@ -47,7 +47,7 @@ def read_directories(dir = "")
       dot_static_files = dot_files - dot_pages
 
       retrieve_posts(dir)
-      retrieve_dirs(base, dir, dot_dirs)
+      retrieve_dirs(dir, dot_dirs)
```

Comment:
We can prefix this as `_base`, but removing it altogether changes the method signature.